src/test/java/uk/gov/ons/fwmt/acceptancetests/smoketest/GatewaySmokeTestSteps.java

```diff
@@ -0,0 +1,166 @@
+package uk.gov.ons.fwmt.acceptancetests.smoketest;
```

Comment:
Can you split the glue code into 2 files. GatewaySmokeTestSteps and LegacyGatewaySmokeTestSteps